.travis.yml

```diff
@@ -86,7 +86,7 @@ before_install:
         brew rm --force $(brew deps --HEAD julia);
         brew install -v --only-dependencies --HEAD julia;
         brew install -v staticfloat/juliadeps/libgfortran;
-        BUILDOPTS="-j3 USECLANG=1 USECCACHE=1 BINARYBUILDER_TRIPLET=x86_64-apple-darwin14";
+        BUILDOPTS="-j3 USECLANG=1 USECCACHE=1 BINARYBUILDER_TRIPLET=x86_64-apple-darwin14 BINARYBUILDER_LLVM_DEBUG=1";
```

Comment:
Does this use a full `Debug` build of LLVM, or just enable assertions? Debug builds would probably be too slow for CI.

Reply:
It uses a `RelWithDebInfo` build + Assertions, my intention was to make assertions give us a bit more useful information, but you are right it might be to slow for CI.

Reply:
Oh ok. I don't think I've ever used a `RelWithDebInfo` build. I would guess it's probably much faster than a full Debug build though.

Reply:
We should call it `BINARYBUILDER_LLVM_RELWITHDEBINFO=1` then; so that we're not mixing metaphors.

Reply:
Hm that is a lot to write..., a full debug build is not really useful in any case.